pmlc/dialect/pxa/transforms/stencil.cc

```diff
@@ -38,102 +34,155 @@ class Stencil {
   // affined.load, mul(f/i), reduce add, terminator.
   const unsigned kNumValidInstrInGemmRegion = 5;
 
+  // The ParallelOp that is being stencelled.
   mlir::AffineParallelOp op;
 
+  // Target tensors, the first two are load, the third is aggregate
+  llvm::SmallVector<Value, kNumTensors> tensors;
+
+  // Target tensors strides, the first two are load, the third is aggregate
+  llvm::SmallVector<mlir::StrideInfo, kNumTensors> tensorsStrides;
+
+  // Found Gemm operation data
+  MulOperationType mulOpType;
+  mlir::AffineLoadOp in1Op;
+  mlir::AffineLoadOp in2Op;
+  AffineReduceOp outOp;
+
 public:
-  explicit Stencil(mlir::AffineParallelOp opIn) : op(opIn) {}
+  explicit Stencil(mlir::AffineParallelOp opIn) : op(opIn){};
```

Comment:
drop the ending semicolon, cpplint catches this.

Can you please install the `pre-commit` hooks?

Reply:
Installed the pre-commit package and cpplint as well.
Ran them and submitted the changes.